### PR TITLE
[artifactory][artifactory-ha] Priority class support, fixes #1164

### LINKED ISSUE #1164: add priorityClass def to artifactory ha nginx deployment
**Is this a request for help?**:

---

**Is this a BUG REPORT or FEATURE REQUEST?** (choose one):

**Version of Helm and Kubernetes**:
helm 2, k8s 1.16

**Which chart**:
artifactory ha

**What happened**:
I cannot provide a priorityClass to nginx pods

**What you expected to happen**:
I can provide the priorityClass in helm value override

**How to reproduce it** (as minimally and precisely as possible):

**Anything else we need to know**:

### PR DESCRIPTION
**Pull request by AshutoshNirkhe**
#### PR Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [ ] Chart Version bumped
- [ ] CHANGELOG.md updated
- [ ] Variables and other changes are documented in the README.md
- [ ] Title of the PR starts with chart name (e.g. `[artifactory]`)

**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #1164

**Special notes for your reviewer**:

### CODE REVIEW

**Review by eldada — APPROVED** [review]

**Comment by AshutoshNirkhe** [comment]
Thanks @eldada 
Can we please get other 2 reviews to finish with this?

**Comment by eldada** [comment]
Can you also copy this change to the `artifactory` chart?

**Comment by AshutoshNirkhe** [comment]
@eldada sure thig, added for 'artifactory' chart as well.

**Comment by eldada** [comment]
Thx @AshutoshNirkhe ! I hope the rest of the reviews will be done early this coming week.

File: stable/artifactory-ha/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # JFrog Artifactory-ha Chart Changelog
 All changes to this chart will be documented in this file.
+## [4.3.3] - Oct 15, 2020
```
Comment (chukka) [inline stable/artifactory-ha/CHANGELOG.md:3]: @AshutoshNirkhe Small correction needed , Add a line space  before the changelog entry

File: stable/artifactory/CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,7 @@
 # JFrog Artifactory Chart Changelog
 All changes to this chart will be documented in this file.
+## [11.3.2] - Oct 15, 2020
```
Comment (chukka) [inline stable/artifactory/CHANGELOG.md:3]: same here

**Review by chukka — CHANGES_REQUESTED** [review]
Please address above review comments

File: stable/artifactory-ha/CHANGELOG.md (hunk 1 above)
Comment (AshutoshNirkhe) [reply stable/artifactory-ha/CHANGELOG.md:4]: @chukka done

File: stable/artifactory/CHANGELOG.md (hunk 2 above)
Comment (AshutoshNirkhe) [reply stable/artifactory/CHANGELOG.md:4]: @chukka done

**Comment by AshutoshNirkhe** [comment]
Added the line spaces in changelog as mentioned.

**Review by chukka — APPROVED** [review]

**Review by rimusz — APPROVED** [review]
LGTM

**Review by rahulsadanandan — APPROVED** [review]
LGTM

**Pull request merged**